apps/dashboard/src/modules/seller/components/POSOverviewTable.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <CardComponent :header="$t('app.Points of Sale')" class="w-full">
+    <DataTable :rows="rows" :value="pointOfSales" :rowsPerPageOptions="[5, 10, 25, 50, 100]" paginator lazy
+               @page="onPage($event)" :totalRecords="totalRecords">
+      <template #header>
+        <div class="flex flex-row align-items-center justify-content-end">
+
+          <Button :label="$t('app.Create')" icon="pi pi-plus" @click="openCreatePOSModal"/>
```

Comment:
This should be in the same row as the header "Points of Sale" in the card. This can be done like in the point of sale info view in the overview card.

Reply:
I was looking at this already, but couldn't found how to easily done it, but the topAction thing is what I need. Fixed!